### PR TITLE
Adds paseo UI

### PR DESCRIPTION
**Pull request by claravanstaden**
Adds config for the Paseo-Sepolia bridge.

### CODE REVIEW

File: web/packages/api/src/environment.ts [hunk 1]
```diff
@@ -306,6 +306,140 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
...
     },
+    paseo_sepolia: {
+        name: "paseo_sepolia",
+        ethChainId: 11155111,
+        locations: [
+            {
+                id: "ethereum",
+                name: "Ethereum",
+                type: "ethereum",
+                destinationIds: ["assethub", "muse"],
```
Comment (alistair-singh) [inline web/packages/api/src/environment.ts:317]:
```suggestion
                destinationIds: ["assethub"],
```

File: web/packages/api/src/environment.ts [hunk 2]
```diff
@@ -306,6 +306,140 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
...
+                        id: "vETH",
+                        address: "0xc3d088842dcf02c13699f936bb83dfbbc6f721ab",
+                        minimumTransferAmount: 1n,
+                    },
+                    {
+                        id: "MUSE",
+                        address: "0xb34a6924a02100ba6ef12af1c798285e8f7a16ee",
+                        minimumTransferAmount: 1n,
+                    },
+                ],
```
Comment (alistair-singh) [inline web/packages/api/src/environment.ts:334]:
```suggestion
                erc20tokensReceivable: [
                    {
                        id: "WETH",
                        address: "0xfff9976782d46cc05630d1f6ebab18b2324d6b14",
                        minimumTransferAmount: 15_000_000_000_000n,
                    },
                ],
```

File: web/packages/api/src/environment.ts [hunk 3]
```diff
@@ -306,6 +306,140 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
...
```
Comment (alistair-singh) [inline web/packages/api/src/environment.ts:365]:
```suggestion
                erc20tokensReceivable: [
                    {
                        id: "WETH",
                        address: "0xfff9976782d46cc05630d1f6ebab18b2324d6b14",
                        minimumTransferAmount: 15_000_000_000_000n,
                    },
                ],
```

File: web/packages/api/src/environment.ts [hunk 4]
```diff
@@ -306,6 +306,140 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
...
+                    maxConsumers: 16,
+                },
+                erc20tokensReceivable: [
+                    {
+                        id: "MUSE",
+                        address: "0xb34a6924a02100ba6ef12af1c798285e8f7a16ee",
+                        minimumTransferAmount: 10_000_000_000_000_000n,
+                    },
+                ],
+            },
```
Comment (alistair-singh) [inline web/packages/api/src/environment.ts:387]: Delete this whole parachain.

**Comment by NunoAlexandre** [comment]
@claravanstaden @alistair-singh do you know when this will be merged? And are we adding at least one parachain to the config so that we can run some tests from/to a parachain?

**Comment by claravanstaden** [comment]
> @claravanstaden @alistair-singh do you know when this will be merged? And are we adding at least one parachain to the config so that we can run some tests from/to a parachain?

Hey @NunoAlexandre! The main holdup is an AH upgrade. The Paseo team has been focused on the relay chain upgrade (to test coretime), and will upgrade BH and AH this week. As soon as the upgrade is done, we can merge this PR and deploy the UI. Are you in the Element room with the Paseo updates? If not, here it is: https://matrix.to/#/#paseo-announcements:matrix.org

I am not sure about adding a parachain... We could add Muse but will check with @alistair-singh.

**Comment by NunoAlexandre** [comment]
> > @claravanstaden @alistair-singh do you know when this will be merged? And are we adding at least one parachain to the config so that we can run some tests from/to a parachain?
> 
> Hey @NunoAlexandre! The main holdup is an AH upgrade. The Paseo team has been focused on the relay chain upgrade (to test coretime), and will upgrade BH and AH this week. As soon as the upgrade is done, we can merge this PR and deploy the UI. Are you in the Element room with the Paseo updates? If not, here it is: https://matrix.to/#/#paseo-announcements:matrix.org
> 
> I am not sure about adding a parachain... We could add Muse but will check with @alistair-singh.

Great, thanks for the reply :) Adding a parachain would be super important to be able to test Para <> Ethereum flows.

**Comment by claravanstaden** [comment]
@alistair-singh I successfully tested Ethereum -> Polkadot direction, so this PR is ready for review. I'll test the other direction as soon as the Sepolia gas price drops.

Screenshot 2024-09-27 at 09 08 29

**Review by alistair-singh — APPROVED** [review]

**Pull request merged**